test-helloworld/README.md

```diff
@@ -95,15 +95,14 @@ fda_status: N/\n"
 In the future this metadata will be searchable if you agree to make the algorithm publicly available.
 
 ### 5 CUDA
-The McCoy Platform supports CUDA 8 and CudNN 5. If you require a different version, please contact us and we will 
-find a way to set it up for you.
+The EnvoyAI Platform supports Nvidia CUDA 8 and CuDNN 5. If your algorithm require a different version, please contact us and we will try to make it available.
 
-Use the __LABEL__ `com.envoyai.nvidia` to toggle on CUDA support.
+Use the __LABEL__ `com.envoyai.nvidia` to enable CUDA support.
 ```Dockerfile
 LABEL com.envoyai.nvidia=true
 ```
 
-You will likely want to base your Docker image off of an NVidia example.
+You will likely want to base your Docker image off of an Nvidia example.

```

Comment:
its actually nVidia, we both got it wrong

Reply:
OK.  I checked their website and it is all uppercase.  Historically it has been as described nVidia.  I believed they have rebranded to uppercase with secondary branding as "nVidia".